### ISSUE TITLE
Guidance steering not turning on.

### ISSUE THREAD

**Issue by cwhandersen** [jason0611/FS19_HeadlandManagement#17]
Hi. 

I've tried your mod and i really like the idea. And I've loved using it for the last 20 mins. But now it suddenly just doesn't work right.
It randomly just started not wanting to turn on Guidance Steering again, and It's pretty annoying.
And i can't figure out why this is happening. 

Before this issue started. I had an issue with Headland Management not turning on. But i found out that it only works when I've selected the tractor in the top left menu.

Hope you can help. 
And thanks for at cool mod

**Comment by cwhandersen** [comment]
The optional setting would be nice anyway. 
I was a using the Claas Axion 920 with a Claas crop sensor and the Lizard LM 72 plow.

**Comment by cwhandersen** [comment]
I changed Speedcontrol because it didn't work right. And then it broke

**Comment by cwhandersen** [comment]
The Speed Control Mod also breaks if i turn it off in the Headland Management menu. And it won't fix itself if i turn it back on

**Comment by cwhandersen** [comment]
It won't turn on if the Headland Management Keybind is set to the same button as Guidance steering. 
I probably should have said that :3

**Comment by jason0611** [comment]
That may be for a while, but it's always random behavior. I can't recommend it.

**Comment by cwhandersen** [comment]
I can see that now. But it was just so convenient to have them on the same button. But I guess it's just a limitation of the game

**Comment by jason0611** [comment]
Yes, this is a limitation of the game. But I'm glad, that this issue seems to be solved. So I have a little time left to return to farming on MVP19... ;-)
Ok, if I close this issue?
